src/MultiValueControl.tsx

```diff
@@ -154,13 +156,16 @@ export class MultiValueControl extends React.Component<IMultiValueControlProps,
     }
     private _filteredOptions = (): string[] => {
         const filter = this.state.filter.toLocaleLowerCase();
-        const opts = this._mergeStrArrays([this.props.options, this.props.selected || []]);
+        const opts = this._mergeStrArrays([this.props.options, this.props.selected || ["aminm"]]);
```

Comment:
It this correct that you have array with "aminm" in it, and before it was an empty array ?

Reply:
good catch